python/py-htseq/Portfile

```diff
@@ -7,29 +7,31 @@ set _name           HTSeq
 set _n              [string index ${_name} 0]
 
 name                py-htseq
-version             0.6.1
+version             0.10.0
 platforms           darwin
 license             GPL-3+
 maintainers         {gmail.com:davide.liessi @dliessi} openmaintainer
 
-description         A framework to process and analyze data from high throughput \
-                    sequencing HTS assays
+description         A framework to process and analyze data from high-throughput \
+                    sequencing (HTS) assays
 long_description    ${description}
 
-homepage            http://www-huber.embl.de/users/anders/${_name}/
+homepage            https://github.com/simon-anders/[string tolower ${_name}]
 master_sites        pypi:${_n}/${_name}
 distname            ${_name}-${version}
 
-checksums           rmd160  034ef4917faabb8659adc89ceb1ab0d94d7703bd \
-                    sha256  9bff6779595ea13c10137dad3bc4ecfd046b0bc620990eaa08920a29dcbd05c5
+checksums           rmd160  1a08aa9cd722f200e8ce2760eaaf879781ca68ff \
+                    sha256  e119af03cf068e981d7e08c4bd6393fd635007b595541943615b72abd676e547 \
+                    size    609412
 
-python.versions     27
+python.versions     27 36
```

Comment:
Shouldn't 3.7 also be added?

Reply:
The dependency pysam is not yet 3.7-compatible in the latest release - https://github.com/pysam-developers/pysam/issues/697

Reply:
Then I think a comment should be added saying that when pysam is available 3.7 should be added.

Reply:
Ok, I'll do it later.